tools/tools.go

```diff
@@ -0,0 +1,8 @@
+// +build tools
```

Comment:
What's this file about?

Reply:
This is how utility package is defined so `go` cli can use them

Reply:
https://github.com/go-modules-by-example/index/blob/master/010_tools/README.md